tools/deploy.sh

```diff
@@ -22,9 +22,36 @@ MARIADB_HOST_IP="${MARIADB_HOST_IP:-"127.0.0.1"}"
 KUBECTL_ARGS="${KUBECTL_ARGS:-""}"
 KUSTOMIZE="go run sigs.k8s.io/kustomize/kustomize/v3"
 RESTART_CONTAINER_CERTIFICATE_UPDATED=${RESTART_CONTAINER_CERTIFICATE_UPDATED:-"false"}
+export NAMEPREFIX=${NAMEPREFIX:-"capm3"}
 
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 
+IRONIC_DEPLOY_FILES="${SCRIPTDIR}/ironic-deployment/basic-auth/default/auth.yaml \
+	${SCRIPTDIR}/ironic-deployment/basic-auth/default/kustomization.yaml \
+	${SCRIPTDIR}/ironic-deployment/basic-auth/keepalived/auth.yaml \
+	${SCRIPTDIR}/ironic-deployment/basic-auth/keepalived/kustomization.yaml \
+	${SCRIPTDIR}/ironic-deployment/basic-auth/tls/default/auth.yaml \
+	${SCRIPTDIR}/ironic-deployment/basic-auth/tls/default/kustomization.yaml \
+	${SCRIPTDIR}/ironic-deployment/basic-auth/tls/keepalived/auth.yaml \
+	${SCRIPTDIR}/ironic-deployment/basic-auth/tls/keepalived/kustomization.yaml \
+	${SCRIPTDIR}/ironic-deployment/certmanager/certificate.yaml \
+	${SCRIPTDIR}/ironic-deployment/default/kustomization.yaml \
+	${SCRIPTDIR}/ironic-deployment/ironic/ironic.yaml \
+	${SCRIPTDIR}/ironic-deployment/keepalived/keepalived_patch.yaml \
+	${SCRIPTDIR}/ironic-deployment/keepalived/kustomization.yaml \
+	${SCRIPTDIR}/ironic-deployment/tls/default/kustomization.yaml \
+	${SCRIPTDIR}/ironic-deployment/tls/default/tls.yaml \
+	${SCRIPTDIR}/ironic-deployment/tls/keepalived/kustomization.yaml \
+	${SCRIPTDIR}/ironic-deployment/tls/keepalived/tls.yaml"
+
+for DEPLOY_FILE in ${IRONIC_DEPLOY_FILES}; do
+  cp "$DEPLOY_FILE" "$DEPLOY_FILE".bak
+  # shellcheck disable=SC2094
+  envsubst <"$DEPLOY_FILE".bak> "$DEPLOY_FILE"
+  # To use sed need to change ${NAMEPREFIX} to NAMEPREFIX to every IRONIC_DEPLOY_FILES
+  #sed -i "s/NAMEPREFIX/${NAMEPREFIX}/g" "${DEPLOY_FILE}"
```

Comment:
are these redundant, if so we should remove them.

Reply:
Fixed! Thanks